desktop/main.ts

```diff
@@ -34,42 +38,44 @@ app.commandLine.appendSwitch('enable-network-information-downlink-max');
 /**
  * Inserts the plain text from the clipboard into the provided browser window's web contents.
  *
- * @param {BrowserWindow} browserWindow - The Electron BrowserWindow instance where the text should be inserted.
+ * @param browserWindow - The Electron BrowserWindow instance where the text should be inserted.
  */
-function pasteAsPlainText(browserWindow) {
+function pasteAsPlainText(browserWindow: BrowserWindow | BrowserView | WebviewTag | WebContents) {
     const text = clipboard.readText();
-    browserWindow.webContents.insertText(text);
+
+    if ('webContents' in browserWindow) {
+        browserWindow.webContents.insertText(text);
+    }
 }
 
 /**
  * Initialize the right-click menu
  * See https://github.com/sindresorhus/electron-context-menu
  *
- * @param {String} preferredLocale - The current user language to be used for translating menu labels.
- * @returns {Function} A dispose function to clean up the created context menu.
+ * @param preferredLocale - The current user language to be used for translating menu labels.
+ * @returns A dispose function to clean up the created context menu.
  */
-
-function createContextMenu(preferredLocale = LOCALES.DEFAULT) {
+function createContextMenu(preferredLocale: Locale = LOCALES.DEFAULT): () => void {
     return contextMenu({
         labels: {
             cut: Localize.translate(preferredLocale, 'desktopApplicationMenu.cut'),
             paste: Localize.translate(preferredLocale, 'desktopApplicationMenu.paste'),
             copy: Localize.translate(preferredLocale, 'desktopApplicationMenu.copy'),
         },
         append: (defaultActions, parameters, browserWindow) => [
-            new MenuItem({
```

Comment:
I've made this update cause `append` function expects to return an array of `MenuItemConstructorOptions`. It seems to work as expected.
![image](https://github.com/Expensify/App/assets/23176449/67053f56-9cee-45f4-bf4c-68e7d14d1d17)